website/docs/r/monitor_diagnostic_setting.html.markdown

```diff
@@ -159,6 +159,8 @@ A `retention_policy` block supports the following:
 * `enabled` - (Required) Is this Retention Policy enabled?
 
 * `days` - (Optional) The number of days for which this Retention Policy should apply.
+ 
+!> **NOTE:** `retention_policy` has been deprecated in favor of `azurerm_storage_management_policy` resource - to learn more information on the deprecation [in the Azure documentation](https://aka.ms/diagnostic_settings_log_retention).
```

Comment:
this should be moved up to line 157 and also copied under line 151

Reply:
fixed, thanks.